ditto/readers/cyme/read.py

```diff
@@ -4539,9 +4668,9 @@ def parse_transformers(self, model):
                 XR = float(transformer_data["xr"])
                 XR0 = float(transformer_data["xr0"])
                 R1 = Z1 / math.sqrt(1 + XR * XR)
-                R0 = Z0 / math.sqrt(1 + XR * XR)
-                X1 = Z1 / math.sqrt(1 + XR0 * XR0)
-                X0 = Z0 / math.sqrt(1 + XR0 * XR0)
+                R0 = Z0 / math.sqrt(1 + XR0 * XR0)
+                X1 = Z1 / math.sqrt(1 + 1 / (XR * XR))
+                X0 = Z0 / math.sqrt(1 + 1 / (XR0 * XR0))
                 complex1 = complex(R0, X0)
                 complex2 = complex(R1, X1)
```

Comment:
Why is this called `complex1`, `complex2`? We can choose better variable names.

Reply:
These have been changed to complex0 and complex1 to reflect zero and positive sequence impedances